src/Faker/Provider/de_AT/Person.php

```diff
@@ -117,4 +117,24 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+    /**
+     * Generates a random austrian Social Security number.
+     * @link https://de.wikipedia.org/wiki/Sozialversicherungsnummer#.C3.96sterreich
+     * @return string
+     */
+    public static function ssn()
+    {
+        $birthdate = (new \DateTime('@' . mt_rand(0, time())))->format('dmy');
+
+        do {
+            $consecutiveNumber = (string) mt_rand(100, 999);
+
+            $verificationNumber = ((int) $consecutiveNumber[0] * 3 + (int) $consecutiveNumber[1] * 7 + (int) $consecutiveNumber[2] * 9
```

Comment:
- Could you put a comment here about the overflow of a day and months (31+ and 12+)
- I think it will look better if you put the calculations on separate lines per digit or per group.

Reply:
- I'm afraid I don't quite understand what you mean with overflow of days/months. Are you talking about the special case Wikipedia mentions?
- I changed the formatting of the calculation.

Reply:
Yes I am talking about special case that Wikipedia mentions :)

Reply:
The day/month overflow is an edge case solution for when there are more Social Security Numbers for a specific day than the about 800 available combinations of the consecutive number and the verification number. In such a case they apparently just replace the birth date with a faulty one (day higher than 31/month higher than 12).

That doesn't change anything about the algorithm to calculate the consecutive number/verification number - it just means, that the birth date may be outside of what is generated by `(new \DateTime('@' . mt_rand(0, time())))->format('dmy');`.

Reply:
Oh okay, German is not my native language so I think I misunderstood in that case. Looks good to me, then.